modules/microshift-config-parameters-table.adoc

```diff
@@ -245,6 +245,10 @@ container_memory_working_set_bytes{container=`router`,namespace=`openshift-ingre
 |IP address block
 |A block of IP addresses from which pod IP addresses are allocated. IPv4 is the default network. Dual-stack entries are supported. The first entry in this field is immutable after {microshift-short} starts. Default range is `10.42.0.0/16`.
 
+|`network.cniPlugin`
+|String
+|Deploys the Open Virtual Networking - Kubernetes (OVN-K) network plugin as the default container network interface (CNI) when empty or set to `"ovnk"`. Valid values are empty value or either `""`, `"ovnk"`, or `"none"`.
```

Comment:
Configuring `none` means we have no CNI, I think we should have some big warning around it. WDYT?

Reply:
yes `none` will mean that the user will have to  provide their own networking driver which will not be managed  by Microshift (upgrades ie)